docs/source/conf.py

```diff
@@ -186,7 +186,7 @@
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = 'images/AiiDA_transparent_logo.png'
```

Comment:
We can then delete `images/AiiDA_transparent_logo.png` because it is no longer used

Reply:
Done